examples/temporal_graph_learning/generate_launch_script_nc.py

```diff
@@ -0,0 +1,85 @@
+import yaml
```

Comment:
i think you can just provide a yaml file directly. please see other examples.

Reply:
Done.